src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.VisualBasic.xaml

```diff
@@ -47,7 +47,11 @@
     </DynamicEnumProperty.DataSource>
     <DynamicEnumProperty.Metadata>
       <NameValuePair Name="VisibilityCondition">
-        <NameValuePair.Value>(not (has-evaluated-value "Application" "OutputType" "Library"))</NameValuePair.Value>
+        <NameValuePair.Value>
+          (and
+            (is-vb)
```

Comment:
Are the properties in this file ever used when we're not in a VB project? I assume not, from the file name.

Reply:
The file shouldn't even exist as far as C# (really, non-VB) projects are concerned.

Reply:
Removing it in next commit, thanks!